api/v1/checks.go

```diff
@@ -287,6 +287,50 @@ func (c *PostgresCheck) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+type MssqlCheck struct {
+	Description string `yaml:"description" json:"description,omitempty"`
+	Driver      string `yaml:"driver" json:"driver,omitempty"`
+	Connection  string `yaml:"connection" json:"connection,omitempty"`
+	Query       string `yaml:"query" json:"query,omitempty"`
+	// Number rows to check for
+	Result int `yaml:"results" json:"results,omitempty"`
+}
+
+// Obfuscate passwords of the form ' password=xxxxx ' from connectionString since
```

Comment:
can we move this function e.g `sanitizeEndpoint` and connectWithDriver into a `sql.go` ? and perhaps rename connectWithDriver to `querySQL` ?